README.rst

```diff
@@ -90,7 +90,64 @@ By default, blockstore is run as an app inside of Open edX. Enable it using the
 
 -------------------------------------------------------
-Running and testing as a separate service (development)
+Development in Devstack
+-------------------------------------------------------
+
+If you want to develop blockstore as a local python package installed in edx-platform, use the following steps:
+
+#. Prerequisite: Have an Open edX `Devstack <https://github.com/openedx/devstack>`_ properly installed and working. Your devstack must use the Nutmeg release of Open edX (or newer) or be tracking the ``master`` branch of ``edx-platform``.
+
+#. To run blockstore as an app inside of Open edX, enable it using the waffle switch `blockstore.use_blockstore_app_api <https://edx.readthedocs.io/projects/edx-platform-technical/en/latest/featuretoggles.html#featuretoggle-blockstore.use_blockstore_app_api>`_.
+
+#. Clone this repo in the ``src/`` directory, a sibling directory of your devstack folder.
+
+#. In your devstack directory, run the following commands in order:
+
+   .. code::
+      
+      # bring up the lms +studio containers
+      make dev.up.large-and-slow
+      # Bring up the studio shell
+      make dev.shell.studio
+      # Unistall the pypi blockstore
+      pip uninstall openedx-blockstore #You will need to confirm the unistall with "Y".
+      #Install your local blockstore.
+      pip install -e /edx/src/blockstore/  
+      exit
+      
+      # do the same in the LMS
+      make dev.shell.lms
+      # Unistall the pypi blockstore
+      pip uninstall openedx-blockstore #You will need to confirm the unistall with "Y".
+      #Install your local blockstore.
+      pip install -e /edx/src/blockstore/  
+      exit
+    
+      #restart the containers.
+      make dev.restart-devserver.lms dev.restart-devserver.studio
+
+      # you can see that your env now installed blockstore from a local directory by running:
+      make dev.shell.lms
+      pip list
+       
+      
+Seeing your changes will sometimes require running make requirements and then restarting the container.
+
+You're probably also going to want to create a collection for using content libraries, using:
+
+   .. code::
+   
+       # Create a "Collection" that new content libraries / xblocks can be created within:
+        docker exec -t edx.devstack.blockstore bash -c "source ~/.bashrc && echo \"from blockstore.apps.bundles.models import Collection; coll, _ = Collection.objects.get_or_create(title='Devstack Content Collection', uuid='11111111-2111-4111-8111-111111111111')\" | ./manage.py shell"
+       # Create an "Organization":
+       docker exec -t edx.devstack.lms bash -c "source /edx/app/edxapp/edxapp_env && echo \"from organizations.models import Organization; Organization.objects.get_or_create(short_name='DeveloperInc', defaults={'name': 'DeveloperInc', 'active': True})\" | python /edx/app/edxapp/edx-platform/manage.py lms shell"
+       
```

Comment:
Some deprecation warnings are expected

Reply:
I think adding that level of hand-holding might just lead to confusion (what if we remove the deprecation warnings, and then people don't see them).